webapp/src/dogma/features/api/apiSlice.ts

```diff
@@ -81,7 +81,7 @@ export const apiSlice = createApi({
         `/v1/projects/${projectName}/repos/${repoName}/files/revisions/${revision}/${filePath}?queryType=IDENTITY`,
     }),
     getHistoryByProjectAndRepoName: builder.query<HistoryDto[], GetFilesByProjectAndRepoName>({
-      query: ({ projectName, repoName }) => `/v1/projects/${projectName}/repos/${repoName}/history`,
+      query: ({ projectName, repoName }) => `/v1/projects/${projectName}/repos/${repoName}/commits/`,
```

Comment:
What do you think of making `getHistoryByProjectAndRepoName` taking `revision` and `maxCommits`? We may browse old commits when `Older` button is clicked.

The default value of `maxCommits` is 100 which would be large for a web application. 
https://github.com/line/centraldogma/blob/e9e82d1c087f1a0ec4f5e5cc9e6609f2130f015e/server/src/main/java/com/linecorp/centraldogma/server/storage/repository/Repository.java#L66
How about setting the value to 20 or 30?

Reply:
Question: how are going to handle the pagination? I don't see `page` param in the API, is it going to be handled in the frontend?

Reply:
I've added a pagination for the table instead. 

<img width="1205" alt="Screen Shot 2023-01-13 at 15 01 21" src="https://user-images.githubusercontent.com/5079588/212268769-ebdc08c3-3271-4c7a-81f1-dc1e4012219b.png">

Reply:
Wow! It is so fast.

Reply:
It looks to handle the pagination on the client side.
Some repositories may have very long histories whose old versions may not be shown without relying on the server-side pagination API. 🤔
To show the histories from head revision, we wouod use negative revision numbers.
For example, the first 20 revisions would be -1 to -20 and the next page should be -21 to -40.

Anyway, the UI design looks good and convenient. This PR is itself good. 👍